### ISSUE TITLE
latest version is using deprecated api

### ISSUE THREAD

**Issue by w19787** [tensorflow/models#2361]
### System information
- **What is the top-level directory of the model you are using**: slim/InceptionV4
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No 
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.3.0
- **Bazel version (if compiling from source)**: no
- **CUDA/cuDNN version**: no
- **GPU model and memory**: no Gpu, 8G memory
- **Exact command to reproduce**:

### How to reproduce
fine turning flower dataset based on pre-trained inception V4 chkpt.
python train_image_classifier.py    --train_dir=${TRAIN_DIR}    --dataset_name=flowers    --dataset_split_name=train    --dataset_dir=${DATASET_DIR}    --model_name=inception_v4    --checkpoint_path=${CHECKPOINT_PATH}    --checkpoint_exclude_scopes=InceptionV4/Logits,InceptionV4/AuxLogits    --trainable_scopes=InceptionV4/Logits,InceptionV4/AuxLogits    --max_number_of_steps=200    --batch_size=32    --learning_rate=0.01    --learning_rate_decay_type=fixed    --save_interval_secs=60    --save_summaries_secs=60    --log_every_n_steps=100    --optimizer=rmsprop    --weight_decay=0.00004    --clone_on_cpu=True

### issue
WARNING:tensorflow:From train_image_classifier.py:466: softmax_cross_entropy (from tensorflow.contrib.losses.python.losses.loss_ops) is deprecated and will be removed after 2016-12-30.
Instructions for updating:
Use tf.losses.softmax_cross_entropy instead. Note that the order of the logits and labels arguments has been changed.
WARNING:tensorflow:From /home/ubuntu/tensorflow/local/lib/python2.7/site-packages/tensorflow/contrib/losses/python/losses/loss_ops.py:398: compute_weighted_loss (from tensorflow.contrib.losses.python.losses.loss_ops) is deprecated and will be removed after 2016-12-30.
Instructions for updating:
Use tf.losses.compute_weighted_loss instead.
WARNING:tensorflow:From /home/ubuntu/tensorflow/local/lib/python2.7/site-packages/tensorflow/contrib/losses/python/losses/loss_ops.py:151: add_loss (from tensorflow.contrib.framework.python.ops.arg_scope) is deprecated and will be removed after 2016-12-30.
Instructions for updating:
Use tf.losses.add_loss instead.
WARNING:tensorflow:From train_image_classifier.py:468: softmax_cross_entropy (from tensorflow.contrib.losses.python.losses.loss_ops) is deprecated and will be removed after 2016-12-30.
Instructions for updating:
Use tf.losses.softmax_cross_entropy instead. Note that the order of the logits and labels arguments has been changed.
WARNING:tensorflow:From /home/ubuntu/tensorflow/local/lib/python2.7/site-packages/tensorflow/contrib/losses/python/losses/loss_ops.py:398: compute_weighted_loss (from tensorflow.contrib.losses.python.losses.loss_ops) is deprecated and will be removed after 2016-12-30.
Instructions for updating:
Use tf.losses.compute_weighted_loss instead.
WARNING:tensorflow:From /home/ubuntu/tensorflow/local/lib/python2.7/site-packages/tensorflow/contrib/losses/python/losses/loss_ops.py:151: add_loss (from tensorflow.contrib.framework.python.ops.arg_scope) is deprecated and will be removed after 2016-12-30.
Instructions for updating:
Use tf.losses.add_loss instead.
INFO:tensorflow:Summary name /clone_loss is illegal; using clone_loss instead.

### finding 

on Aug 31, 2007, the slim update version bring the issue in. I don't know why the old api is using in latest version.
git diff 3d6dc1dd0a93be848a742a1f26ffb51ff96a9d52 42f507f51bd612a9d5dda01672d9460a68b4914f

**Comment by reedwm** [comment]
@sguada can you comment?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
